local-kubernetes/charts/helmfile-wococo.yaml

```
@@ -14,15 +14,14 @@ releases:
     values:
       - ./values-testnet-manager.yml
       - configmap:
-          WS_ENDPOINT: "ws://localwococo-bootnode.wococo:9944"
-          NODE_HTTP_PATTERN: "http://NODE_NAME.wococo:9933"
-          NODE_WS_PATTERN: "ws://NODE_NAME.wococo:9944"
-          TESTNET_MANAGER_CONSENSUS: pos
+          RELAY_CHAIN_RPC_URL: "ws://localwococo-bootnode.wococo:9944"
+          RPC_NODE_URL_PATTERN: "NODE_NAME.wococo:9944"
```

as suggested in issue https://github.com/paritytech/testnet-manager/issues/89 use `POD_NAME`
```suggestion
          RPC_NODE_URL_PATTERN: "POD_NAME.wococo:9944"
```